doc/source/cluster/kubernetes/troubleshooting/troubleshooting.md

```diff
@@ -66,6 +67,15 @@ RayService is a Custom Resource Definition (CRD) designed for Ray Serve. In Kube
 create Ray Serve applications once the RayCluster is ready. If the issue pertains to the data plane, specifically your Ray Serve scripts
 or Ray Serve configurations (`serveConfigV2`), troubleshooting may be challenging. See [rayservice-troubleshooting](kuberay-raysvc-troubleshoot) for more details.
 
+(autoscaler)=
+## Ray Autoscaler
+
+### Ray Autoscaler doesn't scale up, causing the new Ray tasks or actors to remain pending
```

Comment:
Should this be under `Questions?`

Reply:
Updated https://github.com/ray-project/ray/pull/42962/commits/63eb27c4ded5f7878c3ba92f73585a3c61ae4fd0.